src/main/java/uk/gov/hmcts/ccd/domain/model/lau/SearchLog.java

```diff
@@ -19,8 +20,6 @@ public class SearchLog implements Serializable {
 
     public static final long serialVersionUID = 432973322;
 
-    private static final String ANY_SPACE_MATCH = "\\s*";
-
     private String userId;
     private List<String> caseRefs;
```

Comment:
Should we need to initiate with empty list to avoid NPE for the caller?

Reply:
No need, caseRefs can be null.